routers/common/repo.go

```diff
@@ -21,7 +22,63 @@ import (
 	"code.gitea.io/gitea/modules/util"
 )
 
-// ServeBlob download a git.Blob
+func setCommonHeaders(ctx *context.Context, name string, data interface{}) error {
+	// Google Chrome dislike commas in filenames, so let's change it to a space
+	name = strings.ReplaceAll(name, ",", " ")
+
+	// Fixme: ctx.Repo.Repository.IsPrivate is nil
+	// if ctx.Repo.Repository.IsPrivate {
+	// 	ctx.Resp.Header().Set("Cache-Control", "private, max-age=300")
+	// } else {
+	// 	ctx.Resp.Header().Set("Cache-Control", "public, max-age=86400")
+	// }
+	ctx.Resp.Header().Set("Cache-Control", "public, max-age=86400")
```

Comment:
Already commented on this, 86400 is far too long. Would suggest default 300, ideally also exposed in a config variable, very similar to `STATIC_CACHE_TIME`, maybe call it `CONTENT_CACHE_TIME`.

Reply:
Updated